Dockerfile.lokomotive-controller

```diff
@@ -0,0 +1,9 @@
+FROM golang
+
+WORKDIR /usr/src/lokomotive-controller
+
+COPY . /usr/src/lokomotive-controller/
+
+RUN make MOD=vendor install-controller
+
+ENTRYPOINT ["./lokomotive-controller"]
```

Comment:
We should change this to multi part build. We don't want to have golang other tools where app runs.

Reply:
We could even use scratch image :) https://github.com/invidian/validating-admission-webhook-server/blob/master/Dockerfile#L23-L34

Reply:
Done